### ISSUE TITLE
Are layers missing in EfficientNet backbone (block 5)?

### ISSUE THREAD

**Issue by rdmato33** [qubvel/segmentation_models#539]
I recently compared your EfficientNet-B0 backbone with the state-of-art architecture and detected 4 layers were missing in block 5. Only block 5a, 5b and 5c appear when ploting the model (no "skips" are implemented). Where blocks 5d, 5e, 5f and 5g are (MBConv6 5X5) since they appear in the state-of-art architecture (https://ai.googleblog.com/2019/05/efficientnet-improving-accuracy-and.html)?

Here the code used to plot it:
```
from keras.utils.vis_utils import plot_model
plot_model(model, to_file='model_plot.png', show_shapes=True, show_layer_names=True)
```
Thanks a lot.
